src/custom-elements/src/components/date-picker/date-picker.tsx

```diff
@@ -66,6 +68,47 @@ export class DatePicker {
     }
   }
 
+  _checkDateValidity(dateString) {
```

Comment:
@janibolkvadze We are using day.js for date management. Please use [isValid](https://github.com/iamkun/dayjs/blob/dev/docs/en/API-reference.md#validation-isvalid) function for date validation.